packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -22,7 +23,11 @@ export function SignIn(): JSX.Element {
   const {
     components: {
       // @ts-ignore
```

Comment:
Do we still need to ignore this line?

Reply:
Unfortunately it looks like we do still have to ignore it.